scripts/constants.sh

```diff
@@ -34,17 +34,40 @@ else
     SUBNET_EVM_COMMIT="$(git --git-dir="$SUBNET_EVM_PATH/.git" rev-parse HEAD || :)"
 fi
 
+# Don't export them as they're used in the context of other calls
+if [[ -z ${AVALANCHE_VERSION:-} ]]; then
+  # Get module details from go.mod
+  MODULE_DETAILS="$(go list -m "github.com/ava-labs/avalanchego" 2>/dev/null)"
+
+  # Extract the version part
+  AVALANCHE_VERSION="$(echo "${MODULE_DETAILS}" | awk '{print $2}')"
```

Comment:
nit unneeded comment

```suggestion
  AVALANCHE_VERSION="$(echo "${MODULE_DETAILS}" | awk '{print $2}')"
```